server-worker/src/main/java/com/aliyun/emr/rss/service/deploy/worker/FileWriter.java

```diff
@@ -54,15 +54,14 @@ public final class FileWriter extends DeviceObserver {
 
   private final File file;
   private final FileChannel channel;
-  public final File dataRootDir;
```

Comment:
Seems it's not used .

Reply:
Yes, the variable is not used. So I just removed it.